Update azure-stack-diagnostics.md remove -OutputPath

thomasmaurer opened the pull request:
Update azure-stack-diagnostics.md remove -OutputPath

-OutputPath "\\<HLH MACHINE ADDRESS>\c$\logs" not needed (do not work) if -OutputSharePath "<EXTERNAL SHARE ADDRESS>" -OutputShareCredential $using:shareCred is used.

azurecla commented:
Hi __@MSFTMan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (deonhe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

thomasmaurer committed:
Update azure-stack-diagnostics.md remove -OutputPath

Update azure-stack-diagnostics.md remove -OutputPath

-OutputPath "\\<HLH MACHINE ADDRESS>\c$\logs" not needed (do not work) if -OutputSharePath "<EXTERNAL SHARE ADDRESS>" -OutputShareCredential $using:shareCred is used.
